docs/support_table.md

```diff
@@ -0,0 +1,1173 @@
+---
+id: support_table
+title: Support Table
+---
+<!-- {@stylesheet: static/katex.min.css} -->
+<style>
+table tr,
+table td {
+    vertical-align: middle;
+}
+</style>
+
+This is a list of TeX functions, sorted alphabetically. This list includes functions that KaTeX supports and some that it doesn't support. There is a similar page, with functions [sorted by type](./supported.md).
```

Comment:
How about adding a link to detexify?

Reply:
detexify is a nifty utility, but I don't see that it adds much value here.

Reply:
@ronkok Maybe like:
> Know the shape of a character that you want, but not its name? Draw it here!

(http://www.onemathematicalcat.org/MathJaxDocumentation/TeXSyntax.htm)?